src/data/api.tsx

```diff
@@ -0,0 +1,27 @@
+import { request } from '../utils/api';
+import { Recipe } from './types';
+
+export const listRecipes = (name: string): Promise<Array<Recipe>> =>
+    request('GET', '', name).then(
```

Comment:
Usually people just use the `fetch` method directly for the exercise but I see you followed the same approach we have in the web repo. The only difference is our `request` method has the url as part of it, which makes it easier to see where the request is going too (more important when you have more than one api file).